siddhi-extensions/nlp/src/main/java/org/wso2/siddhi/gpl/extension/nlp/NameEntityTypeViaDictionaryStreamProcessor.java

```diff
@@ -45,7 +45,7 @@
     private Dictionary dictionary;
 
     @Override
-    protected List<Attribute> init(AbstractDefinition abstractDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext, boolean outputExpectsExpiredEvents) {
+    protected List<Attribute> init(AbstractDefinition abstractDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext) {
         if (logger.isDebugEnabled()) {
             logger.debug("Initializing Query ...");
```

Comment:
Thanks for the suggestion. Removed this with https://github.com/wso2-gpl/siddhi-extensions/pull/18/commits/79c1f0424ddd03faecf131a635884a8db87cea71 since more useful context information is logged below https://github.com/wso2-gpl/siddhi-extensions/blob/master/nlp/src/main/java/org/wso2/gpl/siddhi/extension/nlp/NameEntityTypeViaDictionaryStreamProcessor.java#L108